### DISCUSSION TITLE
如何设置正文字体的大小？

### DISCUSSION THREAD

**Discussion by TobinZuo** [TheNetAdmin/zjuthesis#165] [General]
我在major/cs/fonts.tex里面只看到了设置字体为宋体，想知道是在哪里设置正文字体大小的？
另一个问题，/bodystyle是什么意思...

**Comment by TheNetAdmin** [comment]
字号

https://github.com/TheNetAdmin/zjuthesis/blob/e1eda9393f1ea08b69da21accafeb32d123ea8a2/zjuthesis.cls#L53

直接在代码库里搜索 `bodystyle` 就可以找到定义

https://github.com/TheNetAdmin/zjuthesis/blob/e1eda9393f1ea08b69da21accafeb32d123ea8a2/config/format/general/layout.tex#L162-L170

**Reply by TobinZuo** [reply]
感谢！
